pki/test.go

```diff
@@ -0,0 +1,32 @@
+package pki
+
+import (
+	"crypto/x509"
+	"testing"
+	"time"
+)
+
+// TestDenylistWithCert sets a new Denylist on the Validator and adds the certificate.
+// This is useful in integrations tests etc.
+func TestDenylistWithCert(t *testing.T, val Validator, cert *x509.Certificate) {
```

Comment:
Change the `Test` prefix, I thought it was a test, but it's a helper

Reply:
now `SetNewDenylistWithCert`